_posts/2024-08-29-release-0_7_1.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title:  "Kroxylicious release 0.7.1"
+date:   2024-08-29 00:00:00 +0000
+author: "Rob Young"
+author_url: "https://www.github.com/robobario"
+categories:  [releases, kroxylicious, record-validation, apicurio-registry]
+---
+
+The Kroxylicious project is very pleased to announce the release of [Kroxylicious 0.7.1](https://github.com/kroxylicious/kroxylicious/releases/tag/v0.7.1). See the [Changelog](https://github.com/kroxylicious/kroxylicious/blob/main/CHANGELOG.md#071) for a list of changes and summary of Deprecations, Changes and Removals.
```

Comment:
I'm tempted to say we are disappointed to announce as its purely a security and p12n release, but happy to go with the traditional wording.